fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -150,21 +158,13 @@ fun OnboardingPage(
 @Composable
 private fun DescriptionText(
     description: String,
-    linkTextState: LinkTextState?,
 ) {
-    if (linkTextState != null && description.contains(linkTextState.text, ignoreCase = true)) {
-        LinkText(
-            text = description,
-            linkTextStates = listOf(linkTextState),
-        )
-    } else {
-        Text(
-            text = description,
-            color = FirefoxTheme.colors.textSecondary,
-            textAlign = TextAlign.Center,
-            style = FirefoxTheme.typography.body2,
-        )
-    }
+    Text(
+        text = description,
+        color = FirefoxTheme.colors.textSecondary,
+        textAlign = TextAlign.Center,
+        style = FirefoxTheme.typography.body2,
+    )
```

Comment:
🔍 Maybe we can just inline this in the upper composable now that the logic is gone